src/app/problem/edit/Description.js

```diff
@@ -1,13 +1,80 @@
 import '../ProblemEditor.css';
 import {SubtitleLine} from "../../commons/titles/TitleLine";
+import React, {useState} from "react";
+import ReactMarkdown from "react-markdown";
+import './Description.css';
+
+function Tabs() {
+    const [toggleState, setToggleState] = useState(1);
+    const [textareaVal, setTextareaVal] = useState('');
+
+    const toggleTab = (index) => {
+        setToggleState(index);
+    }
+
+    const handleTextareaChange = e => {
+        setTextareaVal(e.target.value);
+    };
+
+    return (
+        <div>
+            <div className="tabs is-boxed">
+                <ul>
+                    <li className={toggleState === 1 ? "is-active" : ""}>
```

Comment:
declare two constants above `TAB_WRITE = 1` and `TAB_PREVIEW=2`
and use them instead of the raw numbers